src/setuptools_scm/git.py

```diff
@@ -82,20 +83,23 @@ def fail_on_shallow(wd):
         )
 
 
-def parse(root, describe_command=DEFAULT_DESCRIBE, pre_parse=warn_on_shallow):
+def parse(root, config=None, describe_command=DEFAULT_DESCRIBE, pre_parse=warn_on_shallow):
     """
     :param pre_parse: experimental pre_parse action, may change at any time
     """
+    if not config:
+        config = Configuration(root=root)
+
     if not has_command("git"):
         return
 
-    wd = GitWorkdir.from_potential_worktree(root)
+    wd = GitWorkdir.from_potential_worktree(config.absolute_root)
     if wd is None:
         return
     if pre_parse:
         pre_parse(wd)
 
-    out, err, ret = wd.do_ex(describe_command)
+    out, unused_err, ret = wd.do_ex(describe_command)
```

Comment:
this one is a good find, i believe an extra issue should go open to output those errors, thanks :+1:

Reply:
was probably a flake8 warning in vscode